### PR TITLE
FEATURE New deadband strategies.

### PR DESCRIPTION
**Pull request by Juicestus**
Tested at the meeting but requesting a review in case y'all have any insight or suggestions.

### CODE REVIEW

**Review by Suhas44 — APPROVED** [review]
LGTM 👍

File: util/TorqueMath.java [hunk 1]
```diff
@@ -40,18 +40,50 @@ public static final double constrain(final double n, final double a, final doubl
...
+    }
+
+    /**
+     * Scaled linear deadband
+     *
+     * @param value Value to be deadbanded
+     * @param scale The minimum value
+     */
+    public static final double scaledLinearDeadband(final double value, final double scale) {
         return Math.abs(value) < scale ? 0 : (value - (Math.abs(value) / value) * scale) / (1. - scale);
```
[the remark this answers is not in the document]
Comment (Juicestus) [reply util/TorqueMath.java:61]: Looks like it👍

File: util/TorqueMath.java [hunk 2]
```diff
@@ -40,18 +40,50 @@ public static final double constrain(final double n, final double a, final doubl
...
-    public static final double scaledDeadband(final double value, final double scale) {
+    /**
+     * Power with sign the sign of a.
+     * 
+     * @param a The base
+     * @param b The exponent
+     * @return power with the sign of a.
+     */
+    public static final double powPreserveSign(final double a, final double b) {
+        return a >= 0 ? Math.abs(Math.pow(a, b)) * Math.signum(a) : -powPreserveSign(-a, b);
```
[the remark this answers is not in the document]
Comment (Juicestus) [reply util/TorqueMath.java:51]: That recursion handles decimal exponents.

File: util/TorqueMath.java [hunk 3]
```diff
@@ -40,18 +40,50 @@ public static final double constrain(final double n, final double a, final doubl
...
-    //  * Return deadbanded value n between a and b
-    //  *
-    //  * @param n Value to be deadbanded
-    //  * @param a Value to deadband the value over, the minimum value
-    //  * @param b Value to deadband the value under, the maximum value
-    //  * @return The deadband output
-    //  */
-    public static final double scaledDeadband(final double value, final double scale) {
+    /**
+     * Power with sign the sign of a.
```
[the remark this answers is not in the document]
Comment (Juicestus) [reply util/TorqueMath.java:44]: *power of a with sign of a.

**Pull request merged**